device/bgp/bgpupdatereplay.yaml

```diff
@@ -0,0 +1,33 @@
+components:
+  schemas:
+    Bgp.UpdateReplay:
+      description: >-
+        Ordered BGP Updates ( including both Advertise and Withdraws ) to be sent in the order given in the input to the peer after the BGP session is established.
```

Comment:
Do we need any control/timing gap between the two updates? Similarly, do we need any trigger event to replay the updates again or it will be done on restart of the protocol?

Reply:
Yes , this is kept in a object to allow per update controls such as inter update gap in between the list of updates. Not planning to add it all in one phase but gradually as needed once the basic feature is working.